scripts/deb-preinstall.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+SERVER_HOME=/var/spool/xrootd-monitoring-shoveler
+SERVER_USER=xrootd-monitoring-shoveler
+SERVER_GROUP=xrootd-monitoring-shoveler
+SERVER_NAME="XRootD monitoring shoveler user"
+
+# create user to avoid running server as root
+# 1. create group if not existing
+if ! getent group | grep -q "^$SERVER_GROUP:" ; then
```

Comment:
```suggestion
if ! getent group $SERVER_GROUP >/dev/null; then
```
I tried this command out in a debian:latest container and it seemed to work (returns 0 if the group exists and 2 if it doesn't) so I don't think the grep construction is necessary.

Reply:
Indeed, you are of course correct. I'll adapt this and force-push, always good to save and extra `grep` ;-). It also comes with the advantage that it works with non-enumerable entities. 

For reference, these commands were based on [Debian's packaging guidelines](https://www.debian.org/doc/manuals/securing-debian-manual/bpp-lower-privs.en.html) for "Securing Debian", perhaps I should have questioned these more.

Reply:
Heh, that doc says it's from 2017 but I wouldn't be surprised if that block of code had been copy-pasted for over 20 years...